sql/execution_pipeline.cc

```diff
@@ -251,6 +251,29 @@ doResolveTableName(const ColumnName & fullVariableName,
     return fullVariableName;
 }
 
+std::vector<Utf8String>
+PipelineExpressionScope::
+getTableNames() const
+{
+    std::vector<Utf8String> result;
```

Comment:
It's not known ahead of time, and it's only used in binding.  I don't think it will make enough of difference to be worth the complexity of calculating the final size.